external-crates/move/crates/move-compiler/tests/sui_mode/linter/missing_key.move

```diff
@@ -0,0 +1,70 @@
+module a::m {
+    use sui::object::UID;
+
```

Comment:
We will also need a test for a field with the type `UID`, but where it is some different `UID`, e.g. `todd::object::UID` and `a::b::UID`

Reply:
test case added